python/cudf/cudf/comm/serialize.py

```diff
@@ -6,11 +6,14 @@
 # all (de-)serializtion are attached to cudf Objects:
 # Series/DataFrame/Index/Column/Buffer/etc
 serializable_classes = (
+    cudf.CategoricalDtype,
     cudf.DataFrame,
+    cudf.Index,
+    cudf.MultiIndex,
     cudf.Series,
-    cudf.core.series.Series,
     cudf.core.groupby.groupby.GroupBy,
-    cudf.core.column.column.Column,
+    cudf.core.groupby.groupby._Grouping,
+    cudf.core.column.column.ColumnBase,
```

Comment:
At what point do we need to expose things to Dask versus them being an internal implementation detail? If an object is never generated in a function called by Dask can we safely leave it off this list? I.E. `ColumnBase` is very much an implementation detail of cuDF currently where Dask doesn't need to understand its existence.

Reply:
This is added to make sure subclasses of `ColumnBase` are serialized correctly (like `StringColumn`). It is worth noting that `Column` was already here. `ColumnBase` is merely replacing `Column` in this list.